yeonjae/programmers-0/문자열밀기.js

```diff
@@ -0,0 +1,13 @@
+function solution(A, B) {
+  const wordArr = A.split("");
+  let answer = 0;
+
+  for (let i = 0; i < A.length; i++) {
+    if (wordArr.join("") === B) {
+      return answer;
+    }
+    wordArr.unshift(wordArr.pop());
+    answer++;
+  }
+  return -1;
+}
```

Comment:
```suggestion
function solution(A, B) {
  const wordArr = A.split("");

  for (let i = 0; i < A.length; i++) {
    if (wordArr.join("") === B) {
      return i;
    }
    wordArr.unshift(wordArr.pop());
  }
  return -1;
}
```
- 배열 메서드 unshift, pop을 이용한 풀이 잘봤습니다. 코드를 좀 살펴봤는데 answer 변수는 필요 없을 것 같습니다 ~! if문에서 곧바로 i를 리턴하시면 원하는 결과를 도출할 수 있습니다.
- 다른 사람의 코드지만 아래와 같은 놀라운 풀이도 가능합니다 🙇‍♂️
```js
function solution(A, B) {
  return (B + B).indexOf(A);
}
```

Reply:
헐 이짧은 코드는 뭐죠 ㅇㅅㅇ......!